overlord/devicestate/devicemgr.go

```diff
@@ -987,11 +1011,6 @@ func checkGadgetOrKernel(st *state.State, snapInfo, curInfo *snap.Info, flags sn
 		return nil
 	}
 
-	if release.OnClassic {
```

Comment:
Should we unblock kernel as well right now?  My concern is having a kernel installed, and then later we decide to give real meaning to installing a kernel (putting it in place in the classic system).